Mathlib/Order/Basic.lean

```diff
@@ -206,6 +209,14 @@ alias LE.le.lt_of_ne := lt_of_le_of_ne
 alias LE.le.lt_of_ne' := lt_of_le_of_ne'
 alias LE.le.lt_or_eq := lt_or_eq_of_le
 
+lemma le_imp_eq_iff_le_imp_le : a ≤ b → b = a ↔ a ≤ b → b ≤ a where
```

Comment:
Could you add (unnecessary) parens in the statement, for the sake of readability, in the new statements?